### PR TITLE
add video series and blog series

### PR DESCRIPTION
**Pull request by kgallego203**
## What does this PR do?

- I would like to share two of my pieces of content with this repo - a video series and a blog series both related to Appwrite. These are part of my Octernship tasks.

## Test Plan

N/A

## Related PRs and Issues

N/A

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes.

### CODE REVIEW

**Review by adityaoberai — CHANGES_REQUESTED** [review]
Hey @kgallego203 
Could you please add the blog to the [Mobile Development subcategory under Learning Resources](https://github.com/appwrite/awesome-appwrite#mobile-development)?

The Video is kept in the right place.

**Comment by kgallego203** [comment]
> Hey @kgallego203 Could you please add the blog to the [Mobile Development subcategory under Learning Resources](https://github.com/appwrite/awesome-appwrite#mobile-development)?
> 
> The Video is kept in the right place.

Certainly

**Review by kgallego203 — COMMENTED** [review]
@adityaoberai done 👍🏼 let me know if further changes are needed. thanks

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@
     - [Android](#android)
     - [.NET](#net)
   - [Videos](#videos)
+  - [Blogs](#blogs)
```
Comment (adityaoberai) [inline README.md:32]:
```suggestion
```

Comment (adityaoberai) [reply README.md:32]: Let's remove this and then we should be good 👍

**Review by adityaoberai — CHANGES_REQUESTED** [review]
One final change required

File: README.md (hunk 1 above)
Comment (kgallego203) [reply README.md]: Done 👍🏼

**Review by adityaoberai — APPROVED** [review]
LGTM 👍

**Comment by kgallego203** [comment]
Thank you, @adityaoberai :)

**Pull request merged**